src/features/email/templates/ProjectInvitation.tsx

```diff
@@ -0,0 +1,93 @@
+export const projectInvitationTemplate = (
```

Comment:
The template is a start but would need some work. I'd suggest to use a decent plain text email to get this merged and then create a new issue to add a nice html template.